docs/hub/spaces-storage.md

```diff
@@ -0,0 +1,56 @@
+# Disk usage on Spaces
+
+Every Space comes with a small amount of disk storage. This disk space is ephemeral, meaning its content will be lost if your Space restarts.
+If you need to persist data with a longer lifetime than the Space itself, you can:
+- [Subscribe to a persistent storage upgrade](#persistent-storage)
+- [Use a dataset as a data store](#dataset-storage)
+
+## Persistent storage
+
+You can upgrade your Space to have access to persistent disk space from the **Settings** tab.
+
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-storage-settings.png"/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-storage-settings-dark.png"/>
+</div>
+
+You can choose the storage tier of your choice to get access to disk space that persists across restarts of your Space.
+
+Persistent storage acts like a traditional disk storage, mounted on `/data`.
+
+That means you can `read` and `write to` this storage from your Space as you would with a traditional hard-drive or SSD.
+
+Persistent disk Space can be upgraded to a larger tier at will, though cannot be downgraded to a smaller tier. If you wish to use a smaller persistent storage tier, you must delete your current (larger) storage first.
```

Comment:
```suggestion
Persistent disk space can be upgraded to a larger tier at will, though it cannot be downgraded to a smaller tier. If you wish to use a smaller persistent storage tier, you must delete your current (larger) storage first.
```

Reply:
There's no need to add "it" (in the second clause before cannot) bc it is assumed grammatically but we can add it that's fine with me 🤗